product_docs/docs/pgd/5/appusage/feature-compatibility.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Feature compatibiliy
+navTitle: Feature compatibility
+---
+
+## Server feature/Commit scope interoperability
+
+Not all server features work with all commit scopes. This table shows which interoperate:
+
+<table>
+<thead>
+<tr>
+<td style="text-align:center"></td>
+<td style="text-align:center"> Async<br/><div style="font-size:smaller">(default)</div></td> 
+<td style="text-align:center"><a href="../parallel_apply">Parallel<br/>Apply</a> </td>
+<td style="text-align:center"><a href="../transaction-streaming">Transaction<br/>Streaming</a> </td>
+<td style="text-align:center"><a href="../node_management/decoding_worker">Single<br/>Decoding<br/>Worker</a> </td>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td> <a href="../durability/group-commit">Group Commit</a> </td>
+<td style="text-align:center"> ⛔︎ </td>
+<td style="text-align:center"> ❌ </td>
+<td style="text-align:center">  ❌ </td>
+<td style="text-align:center">  ✅ </td>
+</tr>
+<tr>
+<td> <a href="../durability/camo">CAMO</a> </td>
+<td style="text-align:center"> ⛔︎ </td>
+<td style="text-align:center">  ✅ </td>
+<td style="text-align:center">  ❌ </td>
+<td style="text-align:center">  ❌ </td>
+</tr>
+<tr>
+<td> <a href="../durability/lag-control">Lag Control</a> </td>
+<td style="text-align:center"> ✅ </td>
+<td style="text-align:center"> ✅ </td>
+<td style="text-align:center"> ✅  </td>
+<td style="text-align:center">  ✅ </td>
+</tr>
+<tr>
+<td> <a href="../durability/synchronous_commit">Synchronous Commit</a> </td>
+<td style="text-align:center"> ⛔︎ </td>
+<td style="text-align:center"> ✅ </td>
+<td style="text-align:center">  ✅ </td>
+<td style="text-align:center"> ✅ </td>
+</tr>
+</tbody>
+</table>                           
+
+**Legend**:&nbsp;&nbsp;&nbsp; ⛔︎ Not applicable&nbsp;&nbsp;&nbsp; ❌ Does not interoperate&nbsp;&nbsp;&nbsp; ✅ Interoperates
+
+#### Notes
+
+The Async column in the table represents PGD without a synchronous commit scope in use; Lag Control is not a synchronous commit scope, but a controlling commit scope and is available. 
+
+
+## Commit scope/Commit scope interoperability
+
+Although you cannot mix commit scopes, you can daisy chain them together. This table shows where commit scopes can interoperate through daisy chaining:		
```

Comment:
We talk about "combining rules" instead of "daisy-chaining" elsewhere in the documentation. I suggest sticking to that nomenclature, to avoid confusion.

Reply:
Reference: https://deploy-preview-5147--edb-docs-staging.netlify.app/docs/pgd/latest/durability/commit-scope-rules/#combining-rules

Reply:
```suggestion
Although you cannot mix commit scopes, you can [combine rules](..//durability/commit-scope-rules/#combining-rules) with an `AND` operator. This table shows where commit scopes can be combined:		
```

Reply:
LGTM, note the double slashes in the reference.